doc/content/getting-started/migrating-from-v2/create-application.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Create Application
+weight: 20
+---
+
+## Create a New Application on {{% tts %}}
+
+Create a new application on {{% tts %}} where the end devices will be
+imported. This can be done from by following instructions for [Creating an Application in the Console]({{% ref "getting-started/console/create-application" %}})
+or [Creating an Application using the CLI]({{% ref "getting-started/cli#create-application" %}}).
+
+Using the linked instructions, create an application called `imported-application`.
+
+> **NOTE**: In {{% tts %}}, applications do not have an `AppEUI`. The `AppEUI` is
+> configured per-device.
+
+> **NOTE**: If creating the application using the CLI, do not forget to
+> [link with the Network Server]({{% ref "getting-started/cli#link-application" %}}).
```

Comment:
This should be covered by the aforementioned docs on creating an application with CLI, right?